### PR TITLE
Support json properties

### PR DESCRIPTION
**Pull request by RayeN**
Values of some properties might be stored as JSON, we need to support (dump/load) such properties as well.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/test_serializers.py [hunk 1]
```diff
@@ -54,6 +57,19 @@ def test_converts_properties_to_dict(self):
         self.assertEqual(self.data, {'a': 'b', 'c': 'd'})
 
 
+class DataForJsonValues(SerializerTestCase):
```
Comment (ekoval) [inline tests/test_serializers.py]: No reverse test - for data -> object conversion?

Comment (RayeN) [inline tests/test_serializers.py]: Adding...

**Comment by ekoval** [comment]
Don't you forget to update README and other docstrings (add info about new feature there)?

**Comment by RayeN** [comment]
You're right, fixing

**Comment by ekoval** [comment]
:+1:

**Pull request merged**